user_docs/en/changes.t2t

```diff
@@ -83,6 +83,8 @@ Please refer to [the developer guide https://www.nvaccess.org/files/nvda/documen
 - Instantiating ``winVersion.WinVersion`` objects with unknown Windows versions above 10.0.22000 such as 10.0.25398 returns "Windows 11 unknown" instead of "Windows 10 unknown" for release name. (#15992, @josephsl)
 - Make the AppVeyor build process easier for NVDA forks, by adding configurable variables in appveyor.yml to disable or modify NV Access specific portions of the build scripts. (#16216, @XLTechie)
 - Added a how-to document, explaining the process of building NVDA forks on AppVeyor. (#16293, @XLTechie)
+- A fixed width font is now used in the log viewer and in the python console so that the cursor remain in the same column during vertical navigation.
+It is especially useful to read the error location markers in tracebacks. (#16321,  @CyrilleB79)

```

Comment:
```suggestion
- A fixed width font is now used in the log viewer and in the python console so that the cursor remains in the same column during vertical navigation.
It is especially useful to read the error location markers in tracebacks. (#16321,  @CyrilleB79)
```